llvm/docs/SecurityTransparencyReports.rst

```diff
@@ -2,7 +2,15 @@
 LLVM Security Group Transparency Reports
 ========================================
 
-This page lists the yearly LLVM Security group transparency reports.
+This page lists the yearly LLVM Security Response group transparency reports.
+
+The LLVM Security Response group started out as the LLVM security group, previous
+year's transparency reports keep the original year.
+
+Initially the Chromium issue tracker was used to record issues. This component
+has been archived and is read-only. A llvm-project issue with the SecurityArchive
```

Comment:
Maybe slightly rephrase the last sentence to: "A github llvm/llvm-project issue has been created for each issue in the Chromium issue tracker. All such issues contain an attached PDF with the content of the Chromium issue. All such issues are labeled with the `SecurityArchive <https://github.com/llvm/llvm-project/issues?q=label%3ASecurityArchive>`_ label.

Reply:
I've reworded to get close to the text above.

Reply:
Looks good, thanks!